revolv/project/tests/tests.py

```diff
@@ -242,3 +244,92 @@ class ScrapeTest(TestCase):
     def test_scrape(self):
         result = scrape.delay()
         self.assertTrue(result.successful())
+
+
+class DonationAjaxTestCase(CreateTestProjectMixin, TestUserMixin, TestCase):
+    VALIDATE = 'payment/validate'
+    SUBMIT = 'payment/submit'
+
+    def setUp(self):
+        TestUserMixin.setUp(self)
+        self._send_test_user_login_request()
+        self.project = self.create_test_project()
+        self.project.project_status = Project.ACTIVE
+        self.project.save()
+
+    def _make_valid_payment(self):
+        valid_payment = {
+            'csrfmiddlewaretoken': self.client.cookies['csrftoken'].value,
+            'type': 'visa',
+            'first_name': 'William',
+            'last_name': 'Taft',
+            'expire_month': 6,
+            'expire_year': 2020,
+            'cvv2': '00',
+            'number': '1234123412341234',
+            'amount': '10.00',
+        }
+        resp = self.client.post(
+            self.project.get_absolute_url() + self.VALIDATE,
+            data=valid_payment,
+            HTTP_X_REQUESTED_WITH='XMLHttpRequest'
+        )
+        return resp
+
+    def test_payment_validation_ajax(self):
+        resp = self._make_valid_payment()
+        self.assertEqual(resp.status_code, 200)
+        content = json.loads(resp.content)
+        self.assertTrue(content['valid'])
+
+    def test_invalid_payment_ajax(self):
```

Comment:
Would you mind writing short docstrings that summarize what these methods test? I think that would be good for maintainability purposes